File: Assignment14/program5.c
/////////////////////////////////////////////////////////////////// 
//
// File name : program5.c
// Description : Pattern
// input : iRow = 4  iCol = 4
// Output : 
/*
    1   2   3   4
    2   3   4   5
    3   4   5   6
    4   5   6   7  
*/
// Author : Adinath Santosh Pawar
// Date : 27/5/25
// 
///////////////////////////////////////////////////////////////////

#include<stdio.h>

void Pattern(int iRow,int iCol)
{
    int j = 0;
    int i = 0;

    for(i = 0; i < iRow; i++)
    {    
        for(j = 1; j <= iCol; j++)
        {
           printf("%d\t",i+j);
        }
        printf("\n");
    }

}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("enter number of rows \n");
    scanf("%d",&iValue1);

    printf("enter number of columns \n");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);
    return 0;
}

File: Assignment50/program5.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program5.java
// Description : Pattern printing
// Author : Adinath Santosh Pawar
// Date : 21/8/25           
// 
///////////////////////////////////////////////////////////////////

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        if(iRow != iCol)
        {
            System.out.println("rows and column values should be same");
            return;
        }

        for(int i = 1; i <= iRow; i++)
        {
            for(int j = 1; j <= iCol; j++)
            {
                if(i == 1 || i == iRow || j == 1 || j == iCol || i == j)
                {
                    System.out.print(j + " ");
                }
                else
                {
                    System.out.print("  ");
                }
            }
            System.out.println();
        }
    }
}

class program5
{
    public static void main(String A[])
    {
       Pattern pobj = new Pattern();
       Scanner sobj = new Scanner(System.in);
       int iRow,iCol = 0;

       System.out.print("Enter the number of rows ");
       iRow  = sobj.nextInt();

       System.out.print("Enter the number of columns ");
       iCol  = sobj.nextInt();

       pobj.Pattern(iRow,iCol);
    }
}